pkg/operator/starter.go

```diff
@@ -568,21 +568,30 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		clusterMemberControllerInformers...,
 	)
 
+	go etcdEndpointsController.Run(ctx, 1)
+	go clusterMemberController.Run(ctx, 1)
+
 	go masterNodeInformer.Run(ctx.Done())
 	dynamicInformers.Start(ctx.Done())
 	operatorInformers.Start(ctx.Done())
 	kubeInformersForNamespaces.Start(ctx.Done())
 
+	// wait for all caches to sync before we're starting the majority of controllers
+	kubeInformersForNamespaces.Start(ctx.Done())
```

Comment:
this may be duplicate of line 577

Reply:
yep, thanks. It didn't help, so I'm going to remove that commit again :/